docs/usage/key-concepts/automerge.md

```diff
@@ -8,6 +8,10 @@ description: Learn all about Renovate's automerge functionality here
 Automerging is a Renovate feature that you can use to automate upgrading dependencies.
 When enabled, Renovate will attempt to merge the proposed update once the tests pass.
 
+Keep in mind that Renovate automerges take a bit of time, do not expect Renovate to automerge a PR the second it opens and passes tests.
+Wait for at least an hour before troubleshooting.
```

Comment:
Feel free to suggest a suitable time-frame that people should wait before starting troubleshooting on... 😉